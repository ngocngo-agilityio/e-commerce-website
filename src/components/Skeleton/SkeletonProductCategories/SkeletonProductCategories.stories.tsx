import type { Meta, StoryObj } from '@storybook/react';

// Components
import SkeletonProductCategories from '.';

const meta = {
  title: 'Skeletons/SkeletonProductCategories',
  component: SkeletonProductCategories,
  parameters: {
    layout: 'centered',
  },
  tags: ['autodocs'],
} satisfies Meta<typeof SkeletonProductCategories>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {},
};
