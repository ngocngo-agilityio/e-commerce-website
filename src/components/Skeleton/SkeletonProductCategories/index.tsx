// Libs
import { Box } from '@chakra-ui/react';

const SkeletonProductCategories = (): JSX.Element => {
  return <Box w="600px" h="24px" />;
};

export default SkeletonProductCategories;
