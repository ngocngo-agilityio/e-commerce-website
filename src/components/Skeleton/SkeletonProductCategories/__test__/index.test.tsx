// Libs
import { render } from '@testing-library/react';

// Components
import SkeletonProductCategories from '..';

describe('SkeletonProductCategories component', () => {
  test('should match snapshot for SkeletonProductCategories', () => {
    const { container } = render(<SkeletonProductCategories />);

    expect(container).toMatchSnapshot();
  });
});
