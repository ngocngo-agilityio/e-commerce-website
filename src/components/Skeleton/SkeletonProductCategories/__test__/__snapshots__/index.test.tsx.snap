// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonProductCategories component should match snapshot for SkeletonProductCategories 1`] = `
<div>
  <div
    class="css-d6qow6"
  />
</div>
`;
